src/element.js

```diff
@@ -0,0 +1,5 @@
+import logoSVG from "bundle-text:../logo.svg";
+
+const element = document.createElement("div");
+element.innerHTML = logoSVG;
+export default element.querySelector("svg");
```

Comment:
yep, the code is pretty straight-forward, and is pretty similar to what someone else would use in their project, but by handling the `bundle-text` ourselves, I think we do a lot of good to remove confusion in consuming projects.